Headers/DebugServer2/Types.h

```diff
@@ -106,6 +106,11 @@ struct StopInfo {
     kReasonBreakpoint,
     kReasonSignalStop, // TODO better name
     kReasonTrap,
+#if defined(OS_WIN32)
+    kReasonMemoryError,
+    kReasonMathError,
+    kReasonInstructionError,
```

Comment:
I don't understand what you mean by that.

On Wed, Mar 30, 2016 at 12:33 PM, strager <notifications@github.com> wrote:

> In Headers/DebugServer2/Types.h
> <https://github.com/facebook/ds2/pull/240#discussion_r57949269>:
>
> > @@ -106,6 +106,11 @@ struct StopInfo {
> >      kReasonBreakpoint,
> >      kReasonSignalStop, // TODO better name
> >      kReasonTrap,
> > +#if defined(OS_WIN32)
> > +    kReasonMemoryError,
> > +    kReasonMathError,
> > +    kReasonInstructionError,
>
> Could we make kReasonException which mirrors kReasonTrap (with an
> exception code member variable mirroring signal)?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ds2/pull/240/files/da279dd327af5b2d20a9796c80817d66e80e2b71..dcee5fbd5253932e250a49640779f0f6c30928a2#r57949269>
>



-- 
Stephane Sezer